doc/release/1.17.0-notes.rst

```diff
@@ -99,8 +131,8 @@ identity, it is necessary to also pass in an initial value (e.g.,
 ``initial=np.inf`` for ``np.min``).  For instance, the equivalent of
 ``nansum`` would be, ``np.sum(a, where=~np.isnan(a))``.
 
-Timsort has replaced mergesort as the stable sorting implementation
--------------------------------------------------------------------
+Timesort has replaced mergesort as the stable sorting implementation
```

Comment:
this change is incorrect, the name is Timsort (invented by a guy named Tim:)).

Reply:
how embarrassing, my eyes read timesort down in the text, hence I quickly *fixed* it... sorry for this, I'll fix.